### DISCUSSION TITLE
Publishing blocked

### DISCUSSION THREAD

**Discussion by maxandersen** [quarkiverse/quarkiverse#224] [Team Posts]
Hi,

We now got several reports on published failed. This is due to maven central changing to token only auth. 

We'll get it fixed and give updates asap.

**Comment by maxandersen** [comment]
token and username updated - if publishing does not work please comment and we'll investigate further.

**Comment by holly-cummins** [comment]
Releases are still not working. Are there other steps we're supposed to do on our repos? 

See, for example, https://github.com/quarkiverse/quarkus-pact/actions/runs/9570773133

**Reply by gsmet** [reply]
Your issue is unrelated.

You updated an action and they changed the input they require (from dashed to underscore for whatever reason) https://github.com/quarkiverse/quarkus-pact/commit/db4bb5a2c10836b89046b894c08092a5e67bbfc0 . See https://github.com/crazy-max/ghaction-import-gpg?tab=readme-ov-file#inputs .

But I'm not sure why your repository wasn't part of the big update @gastaldi did for the release process: https://github.com/quarkiverse/quarkus-github-app/blob/main/.github/workflows/release.yml ?

We got rid of this action a while ago.

**Reply by gastaldi** [reply]
Some repositories were created after the big update, I don't remember if that's the case here, but copying from a newer repository should do the job

**Comment by holly-cummins** [comment]
Thanks, both! I’ve got a different error now, so I must have missed something when I copied the new gpg config. I will debug from here … 
--
Holly Cummins
Senior Principal Quarkus Software Engineer, Java Champion
Red Hat  <https://www.redhat.com/>
 <https://www.redhat.com/>
